src/main/java/mate/academy/service/OrderServiceImpl.java

```diff
@@ -4,15 +4,18 @@
 import java.util.List;
 import mate.academy.model.Order;
 import mate.academy.model.Product;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class OrderServiceImpl implements OrderService {
+    private static final Logger logger = LogManager.getLogger(OrderServiceImpl.class);
+
     @Override
     public Order completeOrder(Long userId) {
-        // TODO: add log message about method completeOrder was called
+        logger.debug("completeOrder method was called.");
         List<Product> products = getAllProductsFromShoppingCart(userId);
         Order order = new Order(products, userId);
-        // NOTE: In production ready code this order identifier should be generated by DB
-        // For test purpose we simplify this and return dummy data
+        logger.info("Data was successfully fetched from the DB.");
```

Comment:
The log message 'Data was successfully fetched from the DB.' is misleading because the method 'getAllProductsFromShoppingCart' does not actually fetch data from a database but creates products in memory. Please update the log message to reflect the actual operation or remove it if it's not necessary.